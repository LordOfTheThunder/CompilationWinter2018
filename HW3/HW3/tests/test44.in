struct potato {
	int a;
	bool b;
	byte c;
	int d;
};
int PutToPotato(int a, bool b, byte c, int d, struct potato pot) {
	pot.a = a;
	pot.b = b;
	pot.c = c;
	pot.d = d;
}

void callPutPotato() {
	struct potato pot;
	PutToPotato(313, false, 333b, 22b, pot);
}


void main() { print("Max is cool"); }
