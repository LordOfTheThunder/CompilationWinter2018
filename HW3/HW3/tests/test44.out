---end scope---
a INT -1
b BOOL -2
c BYTE -3
d INT -4
pot struct potato -8
line 16: byte value 333 out of range
